build_cmake/configs/modes.cmake

```diff
@@ -33,6 +42,13 @@ mark_as_advanced(
 )
 
 # Create an UBSAN build variant
+
+# Clang doesn't need to link ubsan, this is only a GCC requirement.
```

Comment:
Hmm, does it require `-static-libsan`? The name makes it sound like it covers both ASan and UBSan.

Reply:
I can't find that many resources on `static-libsan` (seems like a bit of an undocumented feature). It seems to be only related to ASAN since ASAN requires a runtime and UBSAN does not (only discussion I can find https://lists.llvm.org/pipermail/cfe-dev/2018-August/059061.html). 
Testing it out locally, UBSAN works with or without `static-libsan` (as in I can created undefined behavior errors irrespective if I link `static-libsan`)

Reply:
Sounds good! As long as it works ok.

Reply:
👍